R/nse-inject.R

```diff
@@ -400,55 +440,62 @@ NULL
 #' as_label(letters)
 #'
 #' @export
-englue <- function(x) {
-  check_string(x)
+englue <- function(x,
+                   env = caller_env(),
+                   error_call = current_env(),
+                   error_arg = "x") {
```

Comment:
Also mildly surprised this isn't `error_arg(x)`

Reply:
In most cases (tidyeval prog), `caller_arg(x)` will be a string instead of a symbol.